packages/plugin-self-paced-reading/src/index.ts

```diff
@@ -321,13 +321,13 @@ class SelfPacedReadingPlugin implements JsPsychPlugin<Info> {
       }
     }
 
-    // store response
-    let response = {
-      rt_sentence: null,
-      rt_word: null,
-      word: null,
-      word_number: null,
-      sentence: null,
+    // store responses
+    // must be an object, as this is what jsPsych.finishTrial() expects
+    let trial_data = {
+      words: [],
+      rts: [],
+      rts_total: [],
+      sentence: trial.save_sentence ? sentence : null,
```

Comment:
is the `save_sentence` parameter redundant with jsPsych's built in `save_trial_parameters`?

Reply:
This was pre-existing, but I agree it's unnecessary. I removed the `save_sentence` parameter. Incidentally, I'm not convinced it's even necessary to have `sentence` as a data property, since you could easily join the array. I suppose users can decide that for themselves with `save_trial_parameters`.